metadata-db-app/src/cmr/metadata_db/config.clj

```diff
@@ -72,6 +72,11 @@
   from metadata db."
   {:default "cmr_deleted_collection_revision.exchange"})
 
+(defconfig deleted-granule-exchange-name
```

Comment:
Do we always duplicate the exchange name configurations? It seems like we should put them in a namespace somewhere in the message queue lib.